tests/src/system/basic/CLIActionTests.java

```diff
@@ -461,4 +469,71 @@ public void applicationError() throws Exception {
             assertTrue(result, result.contains("ok: deleted"));
         }
     }
+
+    /**
+     * Test creating an action with the "nodejs:default" kind results in a "nodejs:6"
+     * action
+     */
+    @Test
```

Comment:
how about adding this test to scala version instead (this would avoid all the java CLI binding changes above.